Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +204,24 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        empty_tab = Tab.empty
+        tab["poured_from_bottle"] = empty_tab["poured_from_bottle"]
+        tab["HEAD"] = empty_tab["HEAD"]
+        tab["time"] = empty_tab["time"]
+        tab.write
```

Comment:
We probably need to restore these information back to tab file in an `ensure` block.  So we could run `brew bottle` multiple times(considering we will store these info somewhere else).